### PR TITLE
Emilio issue21

### PR DESCRIPTION
**Pull request by Cascadia-EmilioDiaz**
Added photos to the plant information using a ScrollView.

### CODE REVIEW

**Comment by Bansenauer-Cascadia** [comment]
Emilio - - please work through the merge conflicts on your local feature branch and push it again with a new request for the team to review this pull request

**Comment by Bansenauer-Cascadia** [comment]
Also, edit the name of the pull request to "Closes Issue 21" so that the project board will automatically move the issue card to "Review" and close the issue once merged.

**Comment by Bansenauer-Cascadia** [comment]
Request reviews from all of your team members

File: FNCF-PhotoPoints/FNCF-PhotoPoints/PlantInfoViewController.swift [hunk 1]
```diff
@@ -29,12 +30,36 @@ class PlantInfoViewController: UIViewController {
...
+                pictureScrollView.contentSize.width = scrollView.frame.width * CGFloat(i + 1)
+                pictureScrollView.addSubview(imageView)
+                pictureScrollView.delegate = self as? UIScrollViewDelegate
+            }
         }
         
-        plantNameLabel.text = myPlant.name
+        setupImages(imageArray)
+        
+        plantNameLabel.text = plantManager.getPlantByName(name: "Douglas Fir").name
```
Comment (Akohler28) [inline FNCF-PhotoPoints/FNCF-PhotoPoints/PlantInfoViewController.swift]: This view will only ever display douglas fir

File: FNCF-PhotoPoints/FNCF-PhotoPoints/PlantListViewController.swift [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 import UIKit
 
-class PlantListViewController: UITableViewController {
```
Comment (Akohler28) [inline FNCF-PhotoPoints/FNCF-PhotoPoints/PlantListViewController.swift]: needs to be UITableViewController

File: FNCF-PhotoPoints/FNCF-PhotoPoints/PlantInfoViewController.swift (hunk 1 above)
Comment (Cascadia-EmilioDiaz) [reply FNCF-PhotoPoints/FNCF-PhotoPoints/PlantInfoViewController.swift]: The functionality is only on the douglas fir plant, next quarter is where I will adapt it across all

**Pull request merged**